site/usage/applications.md

```diff
@@ -13,7 +13,8 @@ This is an incomplete list of integrations, applications, and extensions of the
 * [Altair](https://github.com/ellisonbg/altair) exposes a Python API for building statistical visualizations that follows Vega-Lite syntax.
 * [Vega for Jupyter notebooks](https://github.com/vega/ipyvega) supports Vega and Vega-Lite charts.
 * [Vega-Lite for Julia](https://github.com/fredo-dedup/VegaLite.jl)
-* [vegaliteR](https://github.com/hrbrmstr/vegalite) provides HTML widgets for Vega-Lite in R.
+* [R ggplot2 "bindings" for Vega-Lite](https://github.com/hrbrmstr/vegalite).
```

Comment:
I look at it and the syntax is closer to us than ggplot2, so I don't know if this is a slightly misleading name?